packages/components/src/VirtualizedList/CellTitle/CellTitleActions.component.js

```diff
@@ -166,6 +166,9 @@ export function CellTitleActionsComponent({
 		<div
 			id={id}
 			className={classNames('main-title-actions-group', theme['main-title-actions-group'])}
+			onKeyDown={e => {
+				e.stopPropagation();
```

Comment:
This allows dropdown gesture without interference with the item to item gesture (up/down arrow)